VERIFICATION.md

```diff
@@ -0,0 +1,106 @@
+Verification Instructions for KEVM
+==================================
+
+We assume that KEVM is installed, and the [K tutorial] has been completed.
```

Comment:
Is this supposed to be a link? You have to provide the URL after the square-braces in parens.

Reply:
Ah, I see you put the links at the bottom of the document.